ui/src/app/base/hooks/base.ts

```diff
@@ -2,7 +2,7 @@ import { useCallback, useEffect, useRef, useState } from "react";
 
 import { NotificationSeverity } from "@canonical/react-components";
 import type { NotificationProps } from "@canonical/react-components";
-import { nanoid } from "nanoid/non-secure";
+import { nanoid } from "@reduxjs/toolkit";
```

Comment:
@petermakowski `nanoid` exported from redux toolkit is the nonsecure version: https://redux-toolkit.js.org/api/other-exports#nanoid.

Reply:
I meant to also say that by switching it back to the reduxjs/toolkit version it gives us one less thing to mock.

Reply:
This was done on purpose, hence the comment:
```jsx
 * @returns non-secure random ID string
```
https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/base/hooks/base.ts#L164

non-secure version is faster and we don't really need enhanced security when generating IDs for the DOM elements.